src/Former/Populator.php

```diff
@@ -148,10 +158,16 @@ protected function parseFieldAsArray($field)
 	public function getAttributeFromModel($model, $attribute, $fallback)
 	{
 		if ($model instanceof Model) {
-			return $model->getAttribute($attribute);
+			// Return fallback if attribute is null
+			$value = $model->getAttribute($attribute);
+			return is_null($value) ? $fallback : $value;
```

Comment:
This is a scary assumption to me; assuming the code:

```
Former::populate($car);
Former::text('year')->value(2014);
```

If `$car->year` is null, do we really want to populate the form with 2014? At the very least, I believe this change will break backwards-compatibility.